README.md

```diff
@@ -44,27 +44,27 @@ Frameworks will be auto-detected when one of the dependencies can be found in th
 | ![](https://raw.github.com/antfu/i18n-ally/master/screenshots/frameworks/i18next.png) [i18next](https://www.i18next.com/) | ✅ | [`i18next`](https://www.npmjs.com/package/i18next) |
 | ![](https://raw.github.com/antfu/i18n-ally/master/screenshots/frameworks/i18ntag.png) [i18nTag](http://i18n-tag.kolmer.net/) | ✅ | [`es2015-i18n-tag`](https://www.npmjs.com/package/es2015-i18n-tag) |
 | ![](https://raw.github.com/antfu/i18n-ally/master/screenshots/frameworks/joomla.png) [Joomla](https://www.joomla.org/) | ✅ | [`joomla/application`](https://docs.joomla.org/Application) |
-| ![](https://raw.github.com/antfu/i18n-ally/master/screenshots/frameworks/chrome.png) [Chrome Extension](https://developer.chrome.com/extensions/i18n) | ✅ | manual config by `"i18n-ally.enabledFrameworks": "chrome-ext"` |
+| ![](https://raw.github.com/antfu/i18n-ally/master/screenshots/frameworks/chrome.png) [Chrome Extension](https://developer.chrome.com/extensions/i18n) | ✅ | manual config with `"i18n-ally.enabledFrameworks": "chrome-ext"` |
 | ![](https://raw.github.com/antfu/i18n-ally/master/screenshots/frameworks/rails.png) [Rails](https://guides.rubyonrails.org/i18n.html#configure-the-i18n-module) | ✅ | [`rails-i18n`](https://github.com/svenfuchs/rails-i18n) |
 | ![](https://raw.github.com/antfu/i18n-ally/master/screenshots/frameworks/laravel.png) [Laravel](https://laravel.com/docs/localization) | ✅ | [`laravel/framework`](https://laravel.com/) |
 | ![](https://raw.github.com/antfu/i18n-ally/master/screenshots/frameworks/angular.png) [Angular](https://angular.io/)(native) | [Vote Here](https://github.com/antfu/i18n-ally/issues/114) |  |
-| 🎎 **Custom**  | ✅ | **For almost any frameworks! [How to...](https://github.com/antfu/i18n-ally/blob/master/docs/custom_framework.md)** |
+| 🎎 **Custom**  | ✅ | **For almost any framework! [How to...](https://github.com/antfu/i18n-ally/blob/master/docs/custom_framework.md)** |
 
 If you would like to make it work for frameworks not listed above, you can:
   - Open a [Framework Support Request](https://github.com/antfu/i18n-ally/issues/new?assignees=&labels=framework+request&template=framework-support-request.md&title=%5BFramework+Request%5D)
-  - Fork and Raise a Pull Request
+  - Fork and raise a Pull Request
   - [Config your custom framework](https://github.com/antfu/i18n-ally/blob/master/docs/custom_framework.md)
 
 
 <h2 align="center"> ⚡ Features </h2>
 
-<h3 align="center"> Inline annotation </h3>
+<h3 align="center"> Inline annotations </h3>
 
 <p align="center">
 <img src='https://github.com/antfu/i18n-ally/blob/master/screenshots/annotation.png?raw=true' width='700px'/>
 </p>
 
-<h3 align="center"> Hover & Direct actions </h3>
+<h3 align="center"> Hover and direct actions </h3>
```

Comment:
Thanks for the fixes! I should really improve my English in formal ways. Though I kinda like the & and capitalized words for emphasizing, wondering what would that make feeling difference to native speakers? Thanks!

Reply:
In formal writing, "&" does not mean exactly the same as "and." Using "&" is rarely correct, but is often used anyway.

Using capitalized words in headings and titles is good, but all the words should be capitalized, except "a", "an", and "and." I changed it because many of the words were not capitalized. Here, let me fix it...